run.sh

```diff
@@ -240,6 +240,12 @@ function parse_arguments() {
                 export POLARION_ADD_SKIPPED="true"
                 shift
                 ;;
+            --report-suffix)
+                if [[ -n "$2" ]]; then
+                    export REPORT_SUFFIX="$2"
```

Comment:
Note, that REPORT_SUFFIX could be defined in one execution and not defined in other.
It means that you need to initiate this environment variable and set it to empty by default.
Take a look at the `variables` file.